<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_SplineOptions</class>
  <widget class="QWidget" name="QG_SplineOptions">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>258</width>
        <height>22</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>4</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>200</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>400</width>
        <height>22</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Spline Options</string>
    </property>
    <layout class="QHBoxLayout">
      <property name="margin">
        <number>1</number>
      </property>
      <item>
        <widget class="QLabel" name="lDegree">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text">
            <string>Degree:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="cbDegree">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <item>
            <property name="text">
              <string>1</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>2</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>3</string>
            </property>
          </item>
        </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="cbClosed">
          <property name="text">
            <string>Closed</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QToolButton" name="bUndo">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text">
            <string>Undo</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <connections>
    <connection>
      <sender>bUndo</sender>
      <signal>clicked()</signal>
      <receiver>QG_SplineOptions</receiver>
      <slot>undo()</slot>
    </connection>
    <connection>
      <sender>cbDegree</sender>
      <signal>activated(QString)</signal>
      <receiver>QG_SplineOptions</receiver>
      <slot>setDegree(QString)</slot>
    </connection>
    <connection>
      <sender>cbClosed</sender>
      <signal>toggled(bool)</signal>
      <receiver>QG_SplineOptions</receiver>
      <slot>setClosed(bool)</slot>
    </connection>
  </connections>
</ui>
